jsonrpsee/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+//! jsonrpsee wrapper crate.
+
+pub use http_client;
```

Comment:
feature gate all internal crates?!

Reply:
You mean like `ws-client`, `ws-server`, `http-client`,`http-server`? Hmm. I suspect most users will want either client or server, but many will want both transports?
Good topic for a conversation with @maciejhirsz too.

Reply:
yepp, my first intention was that `jsonrpsee` will include all and if people want only to use internal crates they can import them individually but it might be weird since `proc-macros` only works from `jsonrpsee crate`. Then the linker would just strip off all unused stuff but the crates will be downloaded still...

I think `server`, `client` and `macros` would be enough and for more fine-granularity then use individual crates.

Reply:
I don't know how many people would use both transports, my hunch is most people (for clients at least) would only need one, but I have no data to back it so I reckon the current granularity is fine. We can always come back and add separate feature flags if need be.